src/pages/GiftAid/GiftAid.js

```diff
@@ -161,9 +161,12 @@ function GiftAid(props) {
             }
           });
         }
-        setFieldValidation({...fieldValidation, [name]: childState});
+        const cloneFieldValidation = fieldValidation;
```

Comment:
if you kept using `fieldValidation` instead of `cloneFieldValidation`, code will behave similarly, as this is not an actual clone, this is just a reference to the object,
If that's alright, just keep using old variable to avoid causing confusion due to variable name!